examples/csv_disklogger.py

```diff
@@ -0,0 +1,31 @@
+# SPDX-FileCopyrightText: 2022 @Skicka for Adafruit Industries / Hakcat
+# Logging data to .CSV file on CircuitPython Disk
+# SPDX-License-Identifier: MIT
+
+## If you get a read-only filesystem error, you must add "storage.remount('/', False)" in boot.py to enable writing to disk
+## Make sure you add a way to reverse this in boot.py! Otherwise your CP device won't show up via USB anymore 
```

Comment:
Looks like this line is too long - breaking it into multiple lines will appease `pylint`!